### PR TITLE
Create Billing Details model

### PR DESCRIPTION
**Pull request by RodrigoLeon-Ksquare**

### CODE REVIEW

**Review by BrandonHernandez-Ksquare — APPROVED** [review]

**Review by ClaudiaDiaz-Ksquare — COMMENTED** [review]
did you add this model to the models array in the index file?

**Review by ClaudiaDiaz-Ksquare — APPROVED** [review]

**Review by BrandonHernandez-Ksquare — DISMISSED** [review]

**Review by BrandonHernandez-Ksquare — DISMISSED** [review]

File: src/models/BillingDetails.model.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  InferAttributes<BillingDetails>,
+  InferCreationAttributes<BillingDetails>
+> {
+  declare id: number;
+  declare ownerName: string;
+  declare billingAddress: string;
+  declare cardNumber: number;
+  declare cvv: number;
+  declare expirationDate: Date;
+  declare clientId: number;
```
Comment (GabrielOsorno-Ksquare) [inline src/models/BillingDetails.model.ts:20]: CreationOptional on id, etc., and snake_case would be great! C:

File: src/models/BillingDetails.model.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        autoIncrement: true,
+        primaryKey: true,
+      },
+      ownerName: DataTypes.STRING,
+      billingAddress: DataTypes.STRING,
+      cardNumber: DataTypes.INTEGER,
+      cvv: DataTypes.INTEGER,
+      expirationDate: DataTypes.DATE,
+      clientId: DataTypes.INTEGER,
+    },
```
Comment (GabrielOsorno-Ksquare) [inline src/models/BillingDetails.model.ts:37]: Same, snake_case, and I think its great to use allowNull or defaultValue properties

Comment (RodrigoLeon-Ksquare) [reply src/models/BillingDetails.model.ts:55]: Hi gabriel I did the changes

**Review by GenaroArredondo-Ksquare — APPROVED** [review]

File: src/models/BillingDetails.model.ts (hunk 2 above)
Comment (RodrigoLeon-Ksquare) [reply src/models/BillingDetails.model.ts:55]: Hi gabriel you´re right. I eliminate the creation optional

**Review by ClaudiaDiaz-Ksquare — APPROVED** [review]

**Review by GabrielOsorno-Ksquare — APPROVED** [review]

**Review by YumilRuedaFlores-Ksquare — APPROVED** [review]

**Review by GenaroArredondo-Ksquare — APPROVED** [review]

**Pull request merged**